Vagrantfile

```diff
@@ -68,50 +68,12 @@ Vagrant.configure("2") do |config|
   #   apt-get update
   #   apt-get install -y apache2
   # SHELL
```

Comment:
Before this part of the docs, there are a lot of auto-generated comments that we could throw out - they make this file harder to read/process, and we can easily recover them either via the Vagrant docs or even get the up-to-date version by running a fresh vagrant init